lib/resources/r4/financial_transaction.yaml

```diff
@@ -0,0 +1,52 @@
+---
+name: Financial Transaction
+field_name_base_url: http://hl7.org/fhir/R4/basic.html
+fields:
+- name: financial-transaction-method
+  description: Payment method codes
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-method
+  binding:
+    description: Payment method codes
+    terminology:
+    - display: v2 Payment Method Code
+      system: https://terminology.hl7.org/CodeSystem/v2-0570
+      info_link: https://www.hl7.org/fhir/v2/0570/index.html
+    - display: Millennium payment method codes
+      system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/20589
+      info_link: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/20589
+
+- name: financial-transaction-amount
```

Comment:
Dont need this

Reply:
Removed https://github.com/cerner/fhir.cerner.com/pull/297/commits/a1a39411fbb35b2ccf79e521ffae5e6ae92d62b9#diff-5125c86cde87f8439cb9d1bb859194e9L18-L28

Reply:
+1